harness.go

```diff
@@ -30,13 +30,6 @@ func yesCreateOpts(timeout time.Duration) CreateOptions {
 	return CreateOptions{Args: []string{"yes"}, Timeout: timeout}
 }
 
-func procMap() map[string]func(context.Context, *CreateOptions) (Process, error) {
-	return map[string]func(context.Context, *CreateOptions) (Process, error){
-		"Blocking": newBlockingProcess,
-		"Basic":    newBasicProcess,
-	}
```

Comment:
I'd leave this as it is so that we can add a different implementation in the future.

Reply:
Done.